README.md

```diff
@@ -110,6 +110,46 @@ If required, you can run individual Django tests as follows:
 We support IE8 and above. We have a free account for testing across
 multiple browsers, thanks to [BrowserStack](www.browserstack.com).
 
+## Functional tests
+
+Functional tests are run using Selenium; the default in your sandbox
+is to do so via a Firefox driver, so you will need Firefox
+installed. Running the functional tests will therefore result in a
+browser being launched.
+
+If you want to run the tests headless (i.e. without launching a
+browser), you have two choices:
+
+### Run the functional tests using Xvbf
+
+To do this, you'll need to install
+[pyvirtualdisplay](http://pyvirtualdisplay.readthedocs.io/en/latest/#installation)
+and Xvbf. This is, apparently, quite hard to do on OS X.
```

Comment:
If I install these libraries, will the tests automatically run headlessly? Or do I need to do something else too?